src/PKSim.Core/Services/OverrideScalingMethod.cs

```diff
@@ -19,7 +20,12 @@ public override double ScaledValueFor(ParameterScaling parameterScaling)
 
       protected override ICommand PerformScaling(ParameterScaling parameterScaling)
       {
-         return _parameterTask.SetParameterValue(parameterScaling.TargetParameter, parameterScaling.SourceParameter.Value);
+         var (sourceParameter, targetParameter) = parameterScaling;
+
+         var updateValueCommand = _parameterTask.SetParameterValue(targetParameter, sourceParameter.Value);
+         var updateValueOriginCommand = _parameterTask.SetParameterValueOrigin(targetParameter, sourceParameter.ValueOrigin);
```

Comment:
This is where I am also updating the value origin. It was actually already working for the one use case of molecule because were were cloning the whole molecule